pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1863,6 +1863,14 @@ public void deleteOfflineTable(String tableName, @Nullable String retentionPerio
         offlineTableName);
     LOGGER.info("Deleting table {}: Removed merge rollup task metadata", offlineTableName);
 
+    // Remove all task metadata
+    // TODO: This only deletes the new ZNode MINION_TASK_METADATA/${offlineTableName} but not the old one
+    //  MINION_TASK_METADATA/<task type>/${offlineTableName}, so we keep the above logic of deleting task
+    //  metadata for specific task types. The above logic should be deleted once the old ZNode is not in
+    //  use any more.
+    MinionTaskMetadataUtils.deleteTaskMetadata(_propertyStore, offlineTableName);
+    LOGGER.info("Deleting all task metadata for table {}", offlineTableName);
```

Comment:
nit: it'd be good to follow the message pattern, check the ones in Line 1859, Line 1864, etc.

Reply:
thanks! fixed